src/components/lesson-item/lesson-item.tsx

```diff
@@ -0,0 +1,20 @@
+import classNames from 'classnames';
+import styles from './lesson-item.module.scss';
+import CommonStyles_module from '../../styles/common-styles.module.scss';
+
+export interface LessonItemProps {
+    title: string;
+    price: string;
+    link: string;
+    className?: string;
+}
+
+export const LessonItem = ({ title, price, link, className }: LessonItemProps) => {
+    return (
+        <div className={classNames(styles.root, className)}>
+            <h1 className={styles.title}>{title}</h1>
+            <span className={CommonStyles_module.price}>{price}</span>
+            <button className={CommonStyles_module.primaryButton} onClick={() => link}>Book Now</button>
```

Comment:
wrap it in `<Link>` component
or at least use react router navigate to

Reply:
Do we want be redirected to the lesson when clicking on the `Card`? I think it'd be better if we get redirected only when we click on the button.

Reply:
that's fine
just use react router to redirect